### ISSUE TITLE
SUPERKEYS: per key type (thumb vs other) "Preferences › Typing Speed"

### ISSUE THREAD

**Issue by massimiliano-della-rovere** [Dygmalab/Bazecor#289]
Bazecor 1.0.0b10.

Having the same "Preferences › Typing Speed" for both the thumb keys (T1..T8) and the normal ones causes you to slow down or hasten when you have to "Hold a SuperKey as a Modifier" (i.e. Hold, Tap&Hold, 2Tap&Hold acting as Shift/Ctrl/Alt/OS) together with at least 1+ keys.

Usually you are a bit slower with the thumb/pinky than index/middle/ring finger when you have to Hold, Tap&Hold, 2Tap&Hold, so it's hard to keep a steady rythm and waiting time when you have to wait for the superkey to be recognized as an "held" modifier.

**Comment by danbuhler** [comment]
I agree. Especially when using a hyperkey. I have to use a 3rd party app for all non-homerow superkeys to work around this.
